chord_metadata_service/restapi/urls.py

```diff
@@ -35,7 +35,7 @@
 router.register(r'diseases', phenopacket_views.DiseaseViewSet)
 router.register(r'metadata', phenopacket_views.MetaDataViewSet)
 router.register(r'biosamples', phenopacket_views.BiosampleViewSet)
-router.register(r'phenopackets', phenopacket_views.PhenopacketViewSet)
+router.register(r'phenopackets', phenopacket_views.PhenopacketViewSet, basename="phenopackets")
```

Comment:
since the basename's been changed to "phenopackets", you will need to fix this in here
https://github.com/CanDIG/katsu/blob/Samarth-Agarwal1/dataset-authz/chord_metadata_service/phenopackets/tests/test_api.py#L219

phenopacket**s**  in `'phenopackets-list'` instead of phenopacket

  ```
  def test_phenopacket(self):
        response = get_response('phenopackets-list', self.phenopacket)
```

Reply:
Or in fact, you can set the basename to `basename="phenopacket"` and then everything should be ok. "phenopacket" is the basename by default when queryset attribute is set.